### PR TITLE
USER_ID from session

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

**Review by viaviaj — APPROVED** [review]
Everything is fine, but I have not found a LoginController that can be used to shorten the code, where is he?

**Comment by margarita-ignatieva** [comment]
> Everything is fine, but I have not found a LoginController that can be used to shorten the code, where is he?

Was in a previous HW

**Review by KseniiaMakarova — COMMENTED** [review]
you have closed this PR
image
you need to reopen it

File: src/main/java/com/internet/shop/controllers/cart/GetAllProductsFromCartController.java [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
     private static final Injector injector = Injector.getInstance("com.internet.shop");
     private ShoppingCartService shoppingCartService =
             (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        req.setAttribute("products", shoppingCartService.getByUserId(USER_ID).getProducts());
+        Long userId = (Long) req.getSession().getAttribute(USER_ID);
+        List<Product> addedProducts = shoppingCartService.getByUserId(userId).getProducts();
```
Comment (KseniiaMakarova) [inline src/main/java/com/internet/shop/controllers/cart/GetAllProductsFromCartController.java]: do you need to set attribute?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/internet/shop/controllers/cart/GetAllProductsFromCartController.java (hunk 1 above)
Comment (margarita-ignatieva) [reply src/main/java/com/internet/shop/controllers/cart/GetAllProductsFromCartController.java]: yes, missed it

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**